src/thumbnailer_near_lossless.cc

```diff
@@ -24,8 +24,14 @@ Thumbnailer::Status Thumbnailer::NearLosslessDiff(WebPData* const webp_data) {
     int curr_size = frame.encoded_size;
     float curr_psnr = frame.final_psnr;
 
-    int min_near_ll = 0;
-    int max_near_ll = 100;
+    // Binary search for near-lossless with the pre-processing values contained
+    // in the 'preprocessing_list' to speed-up the algorithm. It is not
+    // necessary to binary search for all the values in range [0,100]
+    // since actually for near-lossless, the frame size and PSNR are not changed
+    // when the preprocessing increases a small quantity.
+    const int preprocessing_list[6] = {0, 20, 40, 60, 80, 100};
```

Comment:
Can you make this a file constant? Then you can reuse it in NearLosslessEqual, using min_ind=1